code/game/objects/items/weapons/AI_modules.dm

```diff
@@ -265,6 +265,12 @@ AI MODULES
 	origin_tech = "programming=3;materials=4"
 	laws = new /datum/ai_laws/crewsimov
 
+/obj/item/aiModule/crewsimov/cmag_act(mob/user)
+	playsound(src, "sparks", 75, TRUE, SHORT_RANGE_SOUND_EXTRARANGE)
+	to_chat(user, "<span class='warning'>Yellow ooze seeps into [src]'s circuits...</span>")
+	new /obj/item/aiModule/goofsimov(loc)
```

Comment:
can you check to see what would happen if you did this inside of a container

Reply:
TIL why assemblies drop their components at your feet. Fixed